Solution/source/Menu/Routine.cpp

```diff
@@ -85,6 +85,11 @@
 
 //--------------------------------Threads--------------------------------------------------------
 
+int g_MenyooConfigOnceTick = 0;
```

Comment:
I also don't think that the variable **g_MenyooConfigOnceTick** is needed, using **g_MenyooConfigTick** instead should work just good I think :)